### ISSUE TITLE
Spike Required - CoWSwap rate not showing in app.shapeshift.com for routes supported on swap.cow.fi

### ISSUE THREAD

**Issue by tolltalcrypto** [shapeshift/web#5918]
### Overview

Currently, when trying to get rates for certain trading pairs, we are not getting rates that we should be able to see. (they are available via the cow swap interface. see images in references below) 

A spike into why these rates and paths arent showing up for us, so we can provide more rates on these trade pairs. 

@hpayne brought this to our attention here.  https://discord.com/channels/554694662431178782/1186780472676728833/1187104995204665506

### References and additional details

![image](https://github.com/shapeshift/web/assets/94986227/993cc8cf-ad9f-434c-8f64-dde15b8762f2)
![image](https://github.com/shapeshift/web/assets/94986227/58136c9b-6a57-4ce2-a281-9ba0b8f302a9)

https://discord.com/channels/554694662431178782/1186780472676728833/1187104995204665506

### Acceptance Criteria

Spike needed if this can be unlocked and allow for us to offer more of the same trading routes as cowswaps interface.

**Comment by gomesalexandre** [comment]
We had this brought many times, but unfortunately it's not an easy lift @shapeshift/product , see:
https://discord.com/channels/554694662431178782/1128735294540238968/1128765541423792168

```
gomes — 07/12/2023 9:11 PM
Not for native assets yeah, for CoW that's a hyuge lift (i.e https://docs.cow.fi/smart-contracts/eth-flow-contract), for 1inch not sure why we don't have support for it
```

tl;dr, that would mean broadcasting a Tx instead of signing when selling native assets, which breaks all assumptions that CoW swaps are done through signing a message + sending it to CoW API, vs. signing an actual Tx.

Could absolutely be done, but not an easy feat

**Comment by 0xean** [comment]
thanks @gomesalexandre - considering this route is supported by Thorchain, 1inch, lifi, and 0x, I don't see the value in adding support for this. 

would recommend closing but will let @twblack88 make the call.

**Comment by gomesalexandre** [comment]
> thanks @gomesalexandre - considering this route is supported by Thorchain, 1inch, lifi, and 0x, I don't see the value in adding support for this.
> 
> would recommend closing but will let @twblack88 make the call.

100% agree:

- from an effort point of view, this is going to be a huge effort to implement and bring regression risks, with an effort that's relatively on par with implementing EIP-712 for CoW
- the effort would all be for not so much benefit, CoW definitely has the best rates on some routes against 0x/1inch with larger amounts given their MEV protection, but Li.Fi already has tons of routes (and could potentially support native assets CoW routes in the future) which makes it so the chance of CoW being the best on those routes is much lower than what we originally implemented it, as we didn't have Li.Fi
- even if there's an amount/pair that has a slightly better rate with CoW at a specific point in time, we already support the route with other swappers, so it's not like users wouldn't be able to swap, as you noted @0xean, they would only get slightly worse rates than on CoW

**Comment by gomesalexandre** [comment]
Closing as per the previous comment, we already have routes for this and this is going to take forever
